### DISCUSSION TITLE
there is no message or refresh of page after create/delete or any other action

### DISCUSSION THREAD

**Discussion by kkcmadhu** [keycloak/keycloak#19071] [Archive]
Go to create realm page, 
create a relam with just giving a name and click on create button, realm gets created, 
But page does not get refreshed,  there is no notificaiton as well. one cannot figure  out if the realm is created or not..

Like wise go to user screen create a user with mandatory fields and click create user, no refresh, no message.. 
one does not know if the user is really created or not unless we navigate explicitly to users page..

like wise for policy, client etc etc..

Its really hard to use!

Example refer this screen shot, where user creation has happened and responded with 201 as expeted, but ui  still remain in same page without any notification...

![image](https://user-images.githubusercontent.com/997582/225624351-9474a0ed-7ab6-4c8f-87d4-8ffe4e5818f8.png)

**Comment by kkcmadhu** [comment]
this is with keycloak 21.0.1 running behind a proxy

**Comment by kkcmadhu** [comment]
typically thesea re the errors i see in console

![image](https://user-images.githubusercontent.com/997582/225645022-a15b6f5b-4a3d-4f19-bb1f-90940529fd8e.png)

**Comment by ssilvert** [comment]
This should be fixed in the nightly build.  You can try it out at https://github.com/keycloak/keycloak/releases/tag/nightly
